config/initializers/dynamic_params.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.config.after_initialize do
+  Katello::Api::V2::RepositoriesController.include Katello::Concerns::Api::V2::DynamicParams::Repositories
```

Comment:
This can be done now it different places, e.g. in `to_prepare` block in `engine.rb`. This is just for readability? and a default place for similar cases.